core/src/main/java/bisq/core/account/sign/SignedWitness.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * This file is part of Bisq.
+ *
+ * bisq is free software: you can redistribute it and/or modify it
```

Comment:
Bisq

Reply:
There is an "insert a suggestion" feature where you can make easy one liner code changes through github.

Reply:
yeah i know but i think in this case it should be a search and replace operation.